src/components/button/button.types.ts

```diff
@@ -15,7 +15,12 @@ export interface Button {
     icon?: string;
 
     /**
-     * True if the button should be selected. only first button will be selected.
+     * True if the button should be selected. Only first button will be selected.
```

Comment:
too picky, but this should be a separate `chore` commit.

Reply:
I removed it for now :)